types/wlr_output_layout.c

```diff
@@ -405,9 +491,72 @@ void wlr_output_layout_add_auto(struct wlr_output_layout *layout,
 			wlr_log(L_ERROR, "Failed to create wlr_output_layout_output");
 			return;
 		}
+	} else {
+		struct wlr_output_layout_output *l_output_rel;
+
+		wl_list_for_each(l_output_rel, &layout->outputs, link) {
```

Comment:
So if I do `wlr_output_layout_add_auto` twice on the same output, it removes all references to it?